README.md

```diff
@@ -19,19 +19,24 @@ module "admin_tier" {
   github_team                 = "${var.github_team}"
   instance_type               = "${var.instance_type}"
   vpc_id                      = "${var.vpc_id}"
-  security_groups             = ["${var.security_groups}"]
```

Comment:
We might want to add externally defined Security Groups to an admin instance (e.g. open port 80, 443)